fuel-vm/src/tests/metadata.rs

```diff
@@ -622,6 +624,12 @@ fn get_transaction_fields() {
         op::eq(0x10, 0x10, 0x11),
         op::and(0x20, 0x20, 0x10),
 
+        op::movi(0x19, 0x00),
+        op::movi(0x11, owner_idx as Immediate18),
+        op::gtf_args(0x10, 0x19, GTFArgs::PolicyOwner),
```

Comment:
Another question for me to understand here. 
I guess the `PolicyOwner` field is taken from the transaction included in the frame pointer. What is the value of the register `0x19` here used for? From the specs i can infer that it is related to the part of the field that we want to read, but it is not clear if this is the start or end offset or something else. 

Anyway, not related to this PR (also probably this value is ignored since I'd expect the owner value to be fixed size).

Reply:
Same as above I jsut copy pasted the existing. cc @Dentosal